docs/src/main/sphinx/connector/faker.md

```diff
@@ -317,3 +338,41 @@ CREATE TABLE generator.default.customer (
   group_id INTEGER WITH (allowed_values = ARRAY['10', '32', '81'])
 );
 ```
+
```

Comment:
Add a separate section title and move it into the SQL support section

Or at a minimum add a title and link to the section from SQL support 

```
- [](/sql/create-table-as), see also ... 
```

Title could be 

```
Simulating data with CREATE TABLE AS
```

or maybe just

```
Details for CREATE TABLE AS use
```